OpenRA.Mods.Common/Util.cs

```diff
@@ -67,11 +67,26 @@ public static int GetTurnDirection(WAngle facing, WAngle desiredFacing)
 		/// </summary>
 		public static int IndexFacing(WAngle facing, int numFrames)
 		{
+			// 1024 here is the max angle, so we divide the max angle by the total number of facings (numFrames)
```

Comment:
Tbh, I'm not sure what that comment is supposed to tell me which isn't covered by reading `1024 / numFrames`. So I'd remove it.

Reply:
I did not understand this function for some time, because 1024 is also used as the length of a cell (my first intuition, and in general is a fairly generic number being 2^10. In a perfect world all [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) would be commented, but I made a point to comment this one because despite having spent many months with this engine, this took me some time to figure out. You can remove it, but I think that is a step in the wrong direction, particularly as the codebase is already in my view too light on commentary, particular for the trickier parts such as this IndexFacing() function.

Reply:
Having this only here is really odd (especially considering the functions below do the same but don't have it). And putting the comment everywhere is just littering. A better solution would probably be introducing some sort of const `WAngle.MaxAngle` that is used instead of 1024 in a follow-up PR.

Reply:
That's probably the best solution, but in the absence of a constant, I think the comment is worth retaining. And I think the fact that IndexFacing() is the first facing related function in Util.cs, means anyone editing the class is likely to read the comment first even if they aren't modifying that specific function, so it doesn't need to be repeated in all subsequent functions. I am happy to look to replace this with WAngle.MaxAngle or similar in a future PR, if we can leave the comment here for reference until the constant is added.